src/dotnet/Users.Service/EmailsBackend.cs

```diff
@@ -122,30 +122,36 @@ private async Task<DigestParameters> FindUnreadChats(
             if (chat is null)
                 return default;
 
-            var firstUnreadMessages = await ChatsBackend
-                .ListNewEntries(contactId.ChatId, chatPosition.EntryLid, 1, cancellationToken)
+            if (chat.Id.IsPlaceRootChat)
+                return default;
+
+            var unreadMessages = await ChatsBackend
+                .ListNewEntries(contactId.ChatId, chatPosition.EntryLid, 100, cancellationToken)
                 .ConfigureAwait(false);
-            if (firstUnreadMessages.Count == 0)
+            if (unreadMessages.Count == 0)
+                return default;
+
+            var firstUnreadMessage = unreadMessages.FirstOrDefault(x => !x.IsSystemEntry);
+            if (firstUnreadMessage is null)
                 return default;
 
-            var chatEntry = firstUnreadMessages[0];
             var author = await Authors
-                .Get(chatEntry.ChatId, chatEntry.AuthorId, AuthorsBackend_GetAuthorOption.Full, cancellationToken)
+                .Get(firstUnreadMessage.ChatId, firstUnreadMessage.AuthorId, AuthorsBackend_GetAuthorOption.Full, cancellationToken)
                 .ConfigureAwait(false);
             if (author is null)
                 return default;
 
-            var userBeginsAt = TimeZoneInfo.ConvertTimeFromUtc(chatEntry.BeginsAt.ToDateTime(), timeZoneInfo);
+            var userBeginsAt = TimeZoneInfo.ConvertTimeFromUtc(firstUnreadMessage.BeginsAt.ToDateTime(), timeZoneInfo);
             var digestChat = new DigestParameters.DigestChat {
                 Name = chat.Title,
                 Link = UrlMapper.ToAbsolute(Links.Chat(chat.Id)),
                 UnreadCount = maxEntryId - chatPosition.EntryLid,
                 FirstUnreadChatEntry = new DigestParameters.DigestChatEntry {
                     At = DeltaText.Get(userBeginsAt, userNow).Text,
                     AuthorName = author.Avatar.Name,
-                    Text = chatEntry.Content.IsNullOrEmpty()
+                    Text = firstUnreadMessage.Content.IsNullOrEmpty()
                         ? "N/A"
```

Comment:
I have a remark that does not connected directly with this PR.
"N/A" looks ugly. What we can show instead: description of attachment or may be better to look further to the next entry with meaningful content.

Reply:
Agree. We will do it eventually